### DISCUSSION TITLE
Problem setting min and max date on x axis

### DISCUSSION THREAD

**Discussion by supriya-somavnshi** [apexcharts/apexcharts.js#3767] [Q&A]
Hi,

I have 6th month data (9 Oct 2022 to 6 Apr 2023 ) but I want show only 3 Month or 1 month  data on chart .
I am trying to set min date ("2023-01-05") and max date ("2023-04-05") in option xaxis: { min : , max : } on chart .But it is not working .Please suggest me how to set 3 month or one month data on chart .

Reproduce link on  codesandbox : 

https://codesandbox.io/s/broken-worker-6vevbm?file=/src/components/ApexSyncCharts.tsx:203-215

Thanks 
Supriya

**Comment by janstieler** [comment]
Hi,
as  I know there is no min, max.
First decide how you want to write your data in the options. There are different posibilities like data, data series, timeseries …
https://apexcharts.com/javascript-chart-demos/timeline-charts/basic/
